include/fastdds/rtps/writer/ReaderProxy.h

```diff
@@ -405,6 +405,20 @@ class ReaderProxy
         active_ = active;
     }
 
+    /**
+     * @brief Check if the sequence number given has been delivered at least once to the transport layer.
+     *
+     * @param seq_number Sequence number of the change to check.
+     * @param found The sequence number has been found in the list of changes pending to be sent/ack.
+     *              This flag allows to differentiate the case when the change is not found from the one that is found
+     *              but it has not been delivered yet.
+     * @return true if the change has been delivered.
+     * @return false otherwise.
+     */
+    bool has_been_delivered(
```

Comment:
It is not part of the public API even though it is in the `include` folder. At some point it will be moved to `src` folder with the other private header files. Also, this method is indirectly being tested with the public API methods (though maybe it is not with a 100% coverage). Finally, doing some unit tests for this method would require mocking `ChangeForReader_t` class (I have looked and it has not been mocked yet). This implies some work and the feature has some priority.

@EduPonz what do you think? Maybe a follow-up PR can complete the tests.

Reply:
I think we can address this on the final PR to master

Reply:
Finally, a test has been included in b54607603